util/fsdiffer/beforeafter.go

```diff
@@ -0,0 +1,63 @@
+package fsdiffer
+
+import (
+	"fmt"
+	"path/filepath"
+)
+
+// The BeforeAfterFSDiffer is used to generate changes in a given directory
```

Comment:
Since it produced changes between two different points in time:
HistoricalFSDiffer
TemporalFSDiffer

> On Nov 12, 2015, at 17:54, Jonathan Boulle <notifications@github.com> wrote:
> 
> In util/fsdiffer/beforeafter.go:
> 
> > @@ -0,0 +1,63 @@
> > +package fsdiffer
> > +
> > +import (
> > +	"fmt"
> > +	"path/filepath"
> > +)
> > +
> > +// The BeforeAfterFSDiffer is used to generate changes in a given directory
> s/The//
> I don't particularly like the name BeforeAfterFSDiffer, but am struggling to think of an alternative right now..
> 
> ―
> Reply to this email directly or view it on GitHub.
>